src/pages/basic_data/app_admin/central_import/CentralImportDuplicationCheckPage.tsx

```diff
@@ -74,6 +74,20 @@ export const CentralImportDuplicationCheckPage = () => {
               publication={registration}
               setDuplicateSearchFilters={setDuplicateSearchFilters}
             />
+            <Button
+              variant="outlined"
+              color="primary"
+              component={Link}
+              to={`${getImportCandidatePagePath(identifier)}`}>
+              Import
```

Comment:
Vi ønsker at alle tekster vi bruker skal være oversatt. Så i tilfellet her blir det først å legge til `"import": "Importer" i `nbTranslations.json` under `"basic_data.central_import". Så bruke den herfra med dette:
```suggestion
              t('basic_data.central_import.import')
```
Oversetteren fikser for engelsk og nynorsk. @iBuzza kan sikkert gi en liten gjennomgang av flyten der i morgen om du ønsker